src/components/navigation-bar/navigation-bar.spec.jsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { mount, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+import NavigationBar from './navigation-bar';
+
+configure({ adapter: new Adapter() });
+
+describe('navigation-bar Component', () => {
```

Comment:
Let's try and keep our describes consistent, which will make reviewing output easier, e.g.
> Navigation Bar Component